src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/ManualRemoveView.tsx

```diff
@@ -38,122 +43,127 @@ export function ManualRemoveView(props: Props) {
-      user: props.user,
-    },
-    "Scan"
-  );
+  const data: StepDeterminationData = {
+    countryCode: props.countryCode,
+    latestScanData: props.scanData,
+    subscriberBreaches: props.breaches,
+    user: props.user,
+  };
+
+  const stepAfterSkip = getNextGuidedStep(data, "Scan");
 
   return (
-    <div className={styles.main}>
-      <div className={styles.content}>
-        <h3>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-manual-remove-how-to-remove-headline"
-          )}
-        </h3>
-        <ol className={styles.removalStepsList}>
-          <li>
-            <strong>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-title"
-              )}
-            </strong>
-            <span>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-content"
-              )}
-            </span>
-          </li>
-          <li>
-            <strong>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-2-title"
-              )}
-            </strong>
-            <span>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-2-content"
-              )}
-            </span>
-          </li>
-          <li>
-            <strong>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-3-title"
-              )}
-            </strong>
-            <span>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-3-content"
-              )}
-            </span>
-          </li>
-          <li>
-            <strong>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-4-title"
-              )}
-            </strong>
-            <span>
-              {l10n.getString(
-                "fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-4-content"
-              )}
-            </span>
-          </li>
-        </ol>
-      </div>
-      <div className={styles.exposureListing}>
-        <h3 className={styles.questionTooltipWrapper}>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-manual-remove-review-profiles-headline"
-          )}
-        </h3>
-        <div className={styles.exposureList}>
-          {props.scanData.results.map((scanResult, index) => {
-            return (
-              <RemovalCard
-                key={scanResult.onerep_scan_result_id}
-                scanResult={scanResult}
-                isExpanded={index === 0}
-              />
-            );
-          })}
+    <FixView
+      data={data}
+      subscriberEmails={props.subscriberEmails}
+      nextStepHref={stepAfterSkip?.href ?? ""}
+    >
+      <div className={styles.main}>
```

Comment:
All the content in here is the same except for additional indenting, but now wrapped in `<FixView>`, and with the data passed in as props.